tools/packaging/common/envoy_bootstrap.json

```diff
@@ -145,9 +145,44 @@
       "transport_api_version": "V3",
       "grpc_services": [
         {
+        {{ if .call_credentials }}
+        "google_grpc": {
+          "target_uri": "{{ .discovery_address }}",
+          "stat_prefix": "googlegrpcxds",
```

Comment:
won't this break all the stats? I would hope its the same stats as when using xds-grpc, not sure if thats feasible

Reply:
I don't think it is feasible at this point. Even with the 'proxy in agent' it will be tricky ( the real stats will be in agent ). Since this is an option primarily for 'external istiod' and opt-in - it is likely acceptable.